README.md

```diff
@@ -12,7 +12,7 @@ Terraform Controller is a Kubernetes Controller for Terraform, which can address
 - AWS
 - Azure
 - GCP
-- VMware vSphere
+- [VMware vSphere](https://registry.terraform.io/providers/hashicorp/vsphere/latest/docs)
```

Comment:
The official site of VMware vSphere is better:)
After that, let's merge this PR and you can draft another one for enhancing/fixing configuration deletion. Thanks.

Reply:
If a use is curious about what's vSphere, he/him can visit the official site.

Reply:
Sorry ! 
I'm not sure your mean about official site. is this https://registry.terraform.io/providers/hashicorp/vsphere/latest ?
or https://www.vmware.com/ ?

Reply:
https://www.vmware.com/hk/products/vsphere.html ? The title of this section is `Supported Cloud providers`, I think this is the right link.

Reply:
Like https://www.alibabacloud.com/, and https://aws.amazon.com/

Reply:
ok